py-polars/tests/unit/io/test_delta.py

```diff
@@ -33,6 +34,33 @@ def test_scan_delta_version(delta_table_path: Path) -> None:
     assert_frame_not_equal(df1, df2)
 
 
+def test_scan_delta_timestamp_version(tmp_path: Path) -> None:
```

Comment:
Can you tag this with `@pytest.mark.write_disk()`?

Reply:
Done : )